src/test/suite/fixtures/recorded/actions/bringMade.yml

```diff
@@ -0,0 +1,33 @@
+spokenForm: bring made
+languageId: typescript
+command:
+  actionName: bring
+  partialTargets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: m}
+    - type: primitive
+      mark: {type: cursor}
+      selectionType: token
+      position: contents
+      modifier: {type: identity}
+      insideOutsideType: inside
+  extraArgs: []
+marks:
+  default.m:
+    start: {line: 0, character: 6}
+    end: {line: 0, character: 21}
+initialState:
+  documentContents: "const myVariableValue = \"hello\";\r\n"
```

Comment:
Hmm. So I don't think it's worth re-recording everything, but I believe if you don't use line feeds these will get turned into block output and be a bit nicer to read. At least I think that's what's happening here. Check out the example yamls